mkt/submit/tests/test_views.py

```diff
@@ -1011,7 +1010,8 @@ def test_done(self):
         eq_(res.status_code, 200)
 
 
-class TestNextSteps(amo.tests.TestCase):
+class TestNextSteps(TestCase):
+    ##(ej)## Delete below now?
```

Comment:
I think the test is still useful, despite what the comment says (and the comment should be removed).

What do you think, @ngokevin ?